components/frontend/src/fields/DateInput.css

```diff
@@ -1,3 +1,15 @@
-form > div.field {
-    width: 100%;  /* Unfortunately, the date picker does not support fluid, so use this as a work-around */
+.react-datepicker__input-container > input, .react-datepicker-wrapper {
+    width: 100% !important;  /* Unfortunately, the date picker does not support fluid, so use this as a work-around */
```

Comment:
Is the comment still valid, also for this new package?

Reply:
Yup!